R/util.R

```diff
@@ -2244,7 +2244,8 @@ summarize_group <- function(.data, group_cols = NULL, group_funs = NULL, ...){
       }
     }
   }
-  ret
+  # For integer columns (like # of rows, unique) change them to numeric for better usability.
```

Comment:
Can you leave comment on why we do it?

Reply:
Added comment with an example of why we want to do it.